odl/trafos/wavelet.py

```diff
@@ -813,19 +1180,26 @@ def _call(self, coeff):
         arr : `DiscreteLpVector`
```

Comment:
We shouldn't hard-code `DiscreteLpVector`, better to write `domain element` and `range element`.